packages/flutter/lib/src/material/expansion_tile.dart

```diff
@@ -69,6 +70,9 @@ class ExpansionTile extends StatefulWidget {
   /// A widget to display instead of a rotating arrow icon.
   final Widget trailing;
 
+  /// An optional value to initialize the widget as collapsed or expanded.
```

Comment:
The argument is optional, the field isn't. Maybe "Specifies if the list tile is initially expanded (true) or collapsed (false, the default)."

Reply:
Done